docs/examples/c-code/VanDerPol/sources/fmi3Functions.c

```diff
@@ -40,12 +40,12 @@
 // ---------------------------------------------------------------------------
 #define MASK_fmi3GetTypesPlatform        (modelStartAndEnd | modelInstantiated | modelInitializationMode \
 | modelEventMode | modelContinuousTimeMode \
-| modelStepComplete | modelStepFailed | modelStepCanceled \
+| modelStepComplete | modelStepInProgress | modelStepFailed | modelStepCanceled \
```

Comment:
We dropped the support for async dostep with FMI3.0. modelStepInProgess is not possible anymore